### DISCUSSION TITLE
how to get free subscription of azure? is student developer pack is  for both gihub and azure?

### DISCUSSION THREAD

**Discussion by Coder-shank** [github/india#558] [Students]
how to get free subscription of azure? is student developer pack is also for both gihub and azure?

**Comment by bprakashputta** [comment]
You can get free subscription of azure if you have a valid GitHub student developer pack or your college has a verified domain name.

**Comment by TheSuranaverse** [comment]
A group for all to get notified for any `Official GitHub Externship Announcement`, `Discuss & Clear any Doubts`, `Learn & Complete` together.
One-stop place for daily `Events/Workshops` + `Hackathons` + `Internships/Jobs` + `Free Scholarship, Courses, Swags & Goodies` + `24/7 Live Chat and Mentors to help you with your Doubts`  + `24/7 Live Music` & much more...

Link to join: https://discord.gg/ceMXzhfaka
GitHub Solution Forum: https://github.com/github/india/discussions/353

Thank you 🙏
